<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDialog</class>
 <widget class="QDialog" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Decay Cascades</string>
  </property>
  <property name="windowIcon">
   <iconset resource="nuclei.qrc">
    <normaloff>:/edit-find.png</normaloff>:/edit-find.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox
{
    font-weight: bold;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="daughterGroupBox">
     <property name="title">
      <string>Daughter Mass Range</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="minALabel">
        <property name="text">
         <string>Minimal A:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>minA</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="minA">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>299</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="maxA">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>299</number>
        </property>
        <property name="value">
         <number>299</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="maxALabel">
        <property name="text">
         <string>Maximal A:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>maxA</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Parent Constraints</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="parentHalfLifeLabel">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Half Life:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="minParentHalfLifeCheck">
        <property name="text">
         <string>Min.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="HalfLifeSpinBox" name="minParentHalfLife">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
        <property name="value">
         <double>86400.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="maxParentHalfLifeCheck">
        <property name="text">
         <string>Max.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="HalfLifeSpinBox" name="maxParentHalfLife">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
        <property name="value">
         <double>2592000.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gammaGroupBox">
     <property name="title">
      <string>Gamma Constraints</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="minIntensCheck">
        <property name="text">
         <string>Minimal Gamma Intensity:</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="minIntens">
        <property name="suffix">
         <string notr="true"> %</string>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="intermediateGroupBox">
     <property name="title">
      <string>Intermediate Level Constraints</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="intHalfLifeLabel">
        <property name="text">
         <string>Half Life:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="minIntHalfLifeCheck">
        <property name="text">
         <string>Min.</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="HalfLifeSpinBox" name="minIntHalfLife">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000001000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="maxIntHalfLifeCheck">
        <property name="text">
         <string>Max.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="HalfLifeSpinBox" name="maxIntHalfLife">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="value">
         <double>0.000010000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="muValCheck">
        <property name="text">
         <string>Minimal Âµ Value:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QDoubleSpinBox" name="muVal">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QCheckBox" name="muValSkip">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Skip check of this condition if value is unknown</string>
        </property>
        <property name="text">
         <string>Skip if unknown</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="qValCheck">
        <property name="text">
         <string>Minimal Q Value:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QDoubleSpinBox" name="qVal">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QCheckBox" name="qValSkip">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Skip check of this condition if value is unknown</string>
        </property>
        <property name="text">
         <string>Skip if unknown</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="muQAnd">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>If checked only cascades fulfilling the Âµ and Q constraints simultaneously will be added to the search results</string>
          </property>
          <property name="text">
           <string>Require Âµ and Q conditions (AND)</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="muQOr">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>If checked all cascades fulfilling the Âµ or Q constraints or both will be added to the search results</string>
          </property>
          <property name="text">
           <string>Require only one condition (OR)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="anisotropyGroupBox">
     <property name="title">
      <string>Minimal Angular Correlation Coefficients</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="6">
       <widget class="QDoubleSpinBox" name="a42">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QDoubleSpinBox" name="a24">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QDoubleSpinBox" name="a22">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="8">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QRadioButton" name="anisotropyAnd">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If checked only cascades fulfilling all enabled angular correlation coefficient constraints simultaneously will be added to the search result&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Require ALL conditions (AND)</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="anisotropyOr">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If checked all cascades fulfilling at least one of the angular correlation coefficient constraints will be added to the search result&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Require one condition (OR)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="8">
       <widget class="QCheckBox" name="anisotropySkip">
        <property name="toolTip">
         <string>If Î´ or spin values are unknown and this box is checked the cascades will be added to the search results despite it is not possible to test the A&lt;sub&gt;kk&lt;/sub&gt; constraints.</string>
        </property>
        <property name="text">
         <string>Skip checking these conditions if angular correlation coefficients are unknown</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="a22Check">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="a44Label">
        <property name="text">
         <string>A&lt;sub&gt;44&lt;/sub&gt;</string>
        </property>
        <property name="buddy">
         <cstring>a44</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="a22Label">
        <property name="text">
         <string>A&lt;sub&gt;22&lt;/sub&gt;</string>
        </property>
        <property name="buddy">
         <cstring>a22</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="a44">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="a44Check">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="a24Label">
        <property name="text">
         <string>A&lt;sub&gt;24&lt;/sub&gt;</string>
        </property>
        <property name="buddy">
         <cstring>a24</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="3" rowspan="2">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QCheckBox" name="a24Check">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QCheckBox" name="a42Check">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="a42Label">
        <property name="text">
         <string>A&lt;sub&gt;42&lt;/sub&gt;</string>
        </property>
        <property name="buddy">
         <cstring>a42</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HalfLifeSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>HalfLifeSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>minA</tabstop>
  <tabstop>maxA</tabstop>
  <tabstop>minParentHalfLifeCheck</tabstop>
  <tabstop>minParentHalfLife</tabstop>
  <tabstop>maxParentHalfLifeCheck</tabstop>
  <tabstop>maxParentHalfLife</tabstop>
  <tabstop>minIntensCheck</tabstop>
  <tabstop>minIntens</tabstop>
  <tabstop>minIntHalfLifeCheck</tabstop>
  <tabstop>minIntHalfLife</tabstop>
  <tabstop>maxIntHalfLifeCheck</tabstop>
  <tabstop>maxIntHalfLife</tabstop>
  <tabstop>muValCheck</tabstop>
  <tabstop>muVal</tabstop>
  <tabstop>qValCheck</tabstop>
  <tabstop>qVal</tabstop>
  <tabstop>a22Check</tabstop>
  <tabstop>a22</tabstop>
  <tabstop>a44Check</tabstop>
  <tabstop>a44</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="nuclei.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SearchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>579</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SearchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>579</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minParentHalfLifeCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>minParentHalfLife</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxParentHalfLifeCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxParentHalfLife</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minIntensCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>minIntens</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minIntHalfLifeCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>minIntHalfLife</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxIntHalfLifeCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxIntHalfLife</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>muValCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>muVal</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qValCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>qVal</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>a22Check</sender>
   <signal>toggled(bool)</signal>
   <receiver>a22</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>a24Check</sender>
   <signal>toggled(bool)</signal>
   <receiver>a24</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>a42Check</sender>
   <signal>toggled(bool)</signal>
   <receiver>a42</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>a44Check</sender>
   <signal>toggled(bool)</signal>
   <receiver>a44</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>muValCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>muValSkip</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qValCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>qValSkip</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
